app-backend/api/src/main/scala/campaign/Routes.scala

```diff
@@ -216,16 +216,19 @@ trait CampaignRoutes
       ScopedAction(Domain.Campaigns, Action.Clone, None),
       user
     ) {
-      authorizeAuthResultAsync {
-        CampaignDao
-          .authorized(
-            user,
-            ObjectType.Campaign,
-            campaignId,
-            ActionType.View
-          )
-          .transact(xa)
-          .unsafeToFuture
+      authorizeAsync {
+        (for {
+          auth1 <- CampaignDao
+            .authorized(
+              user,
+              ObjectType.Campaign,
+              campaignId,
+              ActionType.View
+            )
+          auth2 <- CampaignDao.isActiveCampaign(campaignId)
+        } yield {
+          auth1.toBoolean && auth2
+        }).transact(xa).unsafeToFuture()
```

Comment:
You might want to do this with the applicative instead of sequentially. I think we have a few examples of that elsewhere, like in `replaceDatasourcePermissions`